README.md

```diff
@@ -1,58 +1,74 @@
+   ```
 
-iv. Start running your own analysis!
+   Note that some form of configuration will be needed so that Nextflow knows how to fetch the required software. This is usually done in the form of a config profile (`YOURPROFILE` in the example command above). You can chain multiple config profiles in a comma-separated string.
 
-```bash
-nextflow run qbic-pipelines/bamtofastq -profile <docker/singularity/conda> --input '*.bam'
-```
+   > - The pipeline comes with config profiles called `docker`, `singularity`, `podman`, `shifter`, `charliecloud` and `conda` which instruct the pipeline to use the named tool for software management. For example, `-profile test,docker`.
+   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+   > - If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
+   > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
 
-See [usage docs](docs/usage.md) for all of the available options when running the pipeline.
+4. Start running your own analysis!
 
-## Documentation
+   <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
+
+   ```console
+   nextflow run nf-core/bamtofastq --input '*.bam' --outdir <OUTDIR> -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
+   ```
 
-The qbic-pipelines/bamtofastq pipeline comes with documentation about the pipeline, found in the `docs/` directory:
+## Documentation
 
-1. [Installation](https://nf-co.re/usage/installation)
-2. Pipeline configuration
-    * [Local installation](https://nf-co.re/usage/local_installation)
-    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-3. [Running the pipeline](docs/usage.md)
-4. [Output and how to interpret the results](docs/output.md)
-5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+The nf-core/bamtofastq pipeline comes with documentation about the pipeline [usage](https://nf-co.re/bamtofastq/usage), [parameters](https://nf-co.re/bamtofastq/parameters) and [output](https://nf-co.re/bamtofastq/output).
```

Comment:
```suggestion
The qbic-pipelines/bamtofastq pipeline comes with documentation about the pipeline [usage](https://github.com/qbic-pipelines/bamtofastq/docs/usage.md), [parameters](https://github.com/qbic-pipelines/bamtofastq/parameters) and [output](https://github.com/qbic-pipelines/bamtofastq/docs/output.md).
```

probably need to double check the links. the parameter rendering may be an issue. can ask robert how he handled this for bactopia